### ISSUE TITLE
  Anbox can't start

### ISSUE THREAD

**Issue by jgxjgx** [anbox/anbox#1960]
$ anbox wait-ready 
[ 2021-12-07 16:42:27] [daemon.cpp:61@Run] [org.freedesktop.DBus.Error.ServiceUnknown] The name org.anbox was not provided by any .service files
$ sudo anbox wait-ready
realpath: '': No such file or directory
realpath: '': No such file or directory
realpath: '': No such file or directory
realpath: '': No such file or directory
realpath: '': No such file or directory
realpath: '': No such file or directory
realpath: '': No such file or directory
realpath: '': No such file or directory
[ 2021-12-07 17:04:58] [daemon.cpp:61@Run] Failed to connect to DBus
$ lsmod | grep -e ashmem_linux -e binder_linux
binder_linux          196608  0
ashmem_linux           20480  0
$ sudo mokutil --sb-state
sudo: mokutil：Command not found

**Comment by Fuseteam** [comment]
uh not enough info can you run `anbox session-manager`?

**Comment by jgxjgx** [comment]
anbox session-manager
[ 2021-12-08 15:49:13] [Renderer.cpp:128@initialize] Failed to select EGL configuration
Stack trace (most recent call last) in thread 13157:
#7    Object "[0xffffffffffffffff]", at 0xffffffffffffffff, in 
#6    Object "/lib/x86_64-linux-gnu/libc.so.6", at 0x7fbce8ac3292, in clone
#5    Object "/lib/x86_64-linux-gnu/libpthread.so.0", at 0x7fbce8e34608, in 
#4    Object "/snap/anbox/213/usr/bin/anbox", at 0x55844d026314, in emugl::Thread::thread_main(void*)
#3    Object "/snap/anbox/213/usr/bin/anbox", at 0x55844cf32358, in RenderThread::main()
#2    Object "/snap/anbox/213/usr/bin/anbox", at 0x55844d021060, in renderControl_decoder_context_t::decode(void*, unsigned long, IOStream*)
#1    Object "/snap/anbox/213/usr/bin/anbox", at 0x55844cfa0562, in 
#0    Object "/snap/anbox/213/usr/bin/anbox", at 0x55844cfa3429, in RendererConfigList::getPackInfo(int*, int*) const
Segmentation fault (Address not mapped to object [(nil)])
段错误 (核心已转储)

**Comment by Fuseteam** [comment]
right-o try this https://github.com/anbox/anbox/issues/1943#issuecomment-972905105

**Comment by jgxjgx** [comment]
$ right -o
Command 'right' not found, did you mean:

  command 'sight' from snap sight (0+git.bdde6c3)
  command 'light' from deb light (1.2.1-3)

**Comment by jgxjgx** [comment]
Need to download?

**Comment by Fuseteam** [comment]
uhh i meant can you try the command i [linked](https://github.com/anbox/anbox/issues/1943#issuecomment-972905105)?

**Comment by jgxjgx** [comment]
Still can't run

**Comment by Fuseteam** [comment]
.......can you give an error message to work with? what does `systemctl --user status anbox-session-manager.service` say?

**Comment by tinumathew12** [comment]
> anbox session-manager

$ anbox session-manager
[ 2021-12-13 08:17:01] [session_manager.cpp:149@operator()] Failed to start as either binder or ashmem kernel drivers are not loaded

pls help !!!
I am a beginner on linux using pop os 
ubuntu version 20.04

**Comment by jgxjgx** [comment]
$ systemctl --user status anbox-session-manager.service
Unit anbox-session-manager.service could not be found.

**Comment by jgxjgx** [comment]
$ sudo /snap/bin/anbox.collect-bug-info 
This script will collect a few interesting things which developers will
need to have a better insight into the Anbox system when something goes
wrong.

PLEASE NOTE: The collected log files may collect information about your
system. Please have a look before you're sending them to anyone!

Collecting anbox log files ... 
+ cp /var/snap/anbox/common/data/system.log /tmp/tmp.inVzKUZhEK
+ cp /var/snap/anbox/common/containers/lxc-monitord.log /tmp/tmp.inVzKUZhEK
cp: cannot stat '/var/snap/anbox/common/containers/lxc-monitord.log': No such file or directory
+ true
+ cp /var/snap/anbox/common/logs/container.log /tmp/tmp.inVzKUZhEK
+ cp /var/snap/anbox/common/logs/console.log /tmp/tmp.inVzKUZhEK
+ /snap/anbox/213/command-anbox.wrapper system-info
+ true
+ [ -e /etc/systemd/system/snap.anbox.container-manager.service ]
+ sudo journalctl --no-pager -u snap.anbox.container-manager.service
+ set +x

DONE!

Do you want to include the kernel log of your system? [y/N]
y
+ dmesg
+ set +x
Generating archive with all log files in /home/jgx/下载 ...
  adding: console.log (deflated 87%)
  adding: container-manager.log (deflated 94%)
  adding: container.log (deflated 89%)
  adding: kernel.log (deflated 82%)
  adding: system-info.log (deflated 16%)
  adding: system.log (deflated 87%)
DONE!

Now please take the ZIP archive generated in your current directory
and attach it to your bug report. Please don't hesitate to have a
look into the archive before you do so to verify you don't leak any
information you don't want!

[Uploading anbox-system-diagnostics-2021-12-13.zip…]()

**Comment by tinumathew12** [comment]
i have decided to change my linux distro. it is a problem of pop os

**Comment by Fuseteam** [comment]
> $ systemctl --user status anbox-session-manager.service Unit anbox-session-manager.service could not be found.

this issue is a duplicate of #1943, please move to that issue and close this one. i''l try and help you over there
that number #1943 is a link please click it and close this issue

**Comment by lnee94** [comment]
try this https://github.com/anbox/anbox/issues/1854

**Comment by Fuseteam** [comment]
@lnee94 now this is a good catch, however the solution posted in #1943 is much more elegant
the author of this issue has trouble with understanding github however so your comment is not helping, please read the actual thread slowly
